lib/src/Uncertainty/Algorithm/WeightedExperiments/LHSResult.cxx

```diff
@@ -39,6 +39,7 @@ CLASSNAMEINIT(LHSResult)
 LHSResult::LHSResult()
   : PersistentObject()
   , optimalIndex_(0)
+  , restart_(0)
   , criteria_(Sample(0, 4))
```

Comment:
we get an initialization order error here

Reply:
Yes got it. Thanks